google/execute.ts

```diff
@@ -14,6 +14,9 @@ export enum ExecuteCommandTypes {
     VolumeRelative = 'action.devices.commands.volumeRelative',
     OpenClose = 'action.devices.commands.OpenClose',
     LockUnlock = 'action.devices.commands.LockUnlock',
+    StartStop = 'action.devices.commands.StartStop',
+    PauseUnpause = 'action.devices.commands.PauseUnpause',
+    Dock = 'action.devices.commands.Dock',
```

Comment:
I don't see the Dock command being handled here. Where is it handled?

Reply:
hmmm its still in the service or i forgot to add it in the pr :grimacing: 
I'll take a look and give you an update. I still need to push the service part

Reply:
Dock command is in the execute service because im handling it as the activatescene command. i'm merging your changes in the service branch and i'll do a pr there also